src/wallet.js

```diff
@@ -67,24 +67,28 @@ const wallet = {
    *
    * @param {string} words Words (separated by space) to generate the HD Wallet seed
    *
-   * @return {Object} {'valid': boolean, 'message': string}
+   * @return {Object} {'valid': boolean, 'message': string, 'words': cleanWords}
    * @memberof Wallet
    * @inner
    */
   wordsValid(words) {
+    let newWordsString = '';
     if (_.isString(words)) {
-      if (words.split(' ').length !== 24) {
+      // 1. Remove one or more spaces (or line breaks) before and after the 24 words
+      // 2. Substitute more then one space (or line break) for a single space
+      newWordsString = words.trim(/\s+/).replace(/\s+/g, ' ');
```

Comment:
Maybe we should include `\r`, `\n`, and `\t`.

Reply:
Resolved in the other comments